examples/classification/plot_lda_qda.py

```diff
@@ -147,6 +147,5 @@ def plot_qda_cov(qda, splot):
     splot = plot_data(qda, X, y, y_pred, fig_index=2 * i + 2)
     plot_qda_cov(qda, splot)
     plt.axis('tight')
-plt.suptitle('Linear Discriminant Analysis vs Quadratic Discriminant'
```

Comment:
Why is this removed?

Reply:
I thought it was obvious. I feel it created clutter, and it doesn't play well with `plt.tight_layout()`

Reply:
It might be obvious when you're focused on this matter, but when you're reviewing hundreds of pull requests, perhaps less so.

Reply:
kk, will add back

Reply:
Oh I misunderstood. I thought you were saying that the reason for its removal was obvious, not that the title was redundant. Yes, looking at it now, I'm happy for its removal! Sorry....